features/accessibility.feature

```diff
@@ -5,18 +5,11 @@ Feature: Accessibility
 
 Scenario Outline: Document formats
   Given the customer prefers to receive documentation in <format> format
-  When we send them documentation
+  When we send them their record of guidance
```

Comment:
If we want to do this it should be a different PR as it is not related to removing Williams Lee and would affect multiple features.

The change I have made which is a simple rationalisation of the steps as `we send them documentation` was just calling `we send them their record of guidance`